sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/CancellationTokenSourceExtensions.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+
+namespace Azure.Messaging.ServiceBus
+{
+    internal static class CancellationTokenSourceExtensions
+    {
+        public static void CancelAfter(this CancellationTokenSource cancellationTokenSource,
```

Comment:
I wonder if we should call this something like "CancelOnLockExpiration" or similar, so that the intent is clear.

Reply:
Yeah, I was also thinking about renaming it to something like CancelAfterLockExpired